source/includes/steps-starting-vsce-individual-fields.yaml

```diff
@@ -411,6 +382,61 @@ content: |
            tunnel. |vsce| cannot establish a connection
            to multiple servers across the same SSH tunnel.
 ---
+title: "Specify replica set and security connection options."
+level: 4
+optional: true
+ref: connect-additional-options
+content: |
+  You can specify the following security connection options:
+
+  .. list-table::
+     :header-rows: 1
+
+     * - Option
+       - Description
+
+     * - :guilabel:`Replica Set Name`
```

Comment:
This looks good for now, however, in our next release we're updating the replica set name location to the `General Tab` and we'll have three connection type tabs: `Standalone`, `Replica Set, and `SRV Record`. Should I just open a new docs ticket when that's in or do we want to include those changes here?

Gif of new position:
![connecting to replica set wiht different hosts and ports](https://user-images.githubusercontent.com/1791149/102105232-29120880-3dfd-11eb-891c-434ff92ba0b4.gif)

Reply:
Yes, as @jeff-allen-mongo mentioned, can you please open another doc ticket for this? Thanks!